frontend/src/summary.pug

```diff
@@ -64,10 +64,11 @@
         )
           input.summary-picker__checkbox.mui-checkbox(
             type="checkbox",
-            v-model="isMergeGroup",
-            v-bind:disabled="filterGroupSelection === 'groupByNone'"
+            v-bind:checked="isAllGroupsMerged()",
+            v-bind:disabled="filterGroupSelection === 'groupByNone'",
+            v-on:click="handleMergeGroupCheckboxClicked()"
```

Comment:
dont need the click function. can just use vue set and get in your computed property and bind the model to isAllGroupsMerged